rosidl_runtime_py/convert.py

```diff
@@ -144,7 +144,7 @@ def to_string(val, field_type=None):
     result = ''
 
     # We rely on __slots__ retaining the order of the fields in the .msg file.
```

Comment:
Actually, we should update this comment as well to say `get_fields_and_field_types`.

Reply:
Done.